source/developers-guide/event-guide/index.md

```diff
@@ -0,0 +1,185 @@
+```php
+public function onApplicationStart(Enlight_Event_EventArgs $args) {
+	die("This is only a test");
+}
+```
+
+If you want a detailed guide of how to use an event you can take a look at the [plugin quick start guide](https://developers.shopware.com/developers-guide/plugin-quick-start/#hooking-into-the-system).
+
+## Global lifecycle events
+`Global lifecycle events` are events which will be fired with every lifecycle of a request. The `global lifecycle events` 
+are defined in the class `Enlight_Controller_Front` which is located in `engine/Library/Enlight/Controller/Front.php`.
+
+Usually these are quite technical events like
+* Starting the routing
+* Routing ended
+* Starting dispatch loop
+
+This list will give you an overview of all global lifecycle events in the order they will be triggered. 
+
+* `Enlight_Controller_Front_StartDispatch`
+* `Enlight_Controller_Router_Default`
+* `Enlight_Controller_Dispatcher_Default`
+* `Enlight_Controller_Request_RequestHttp`
+* `Enlight_Controller_Response_ResponseHttp`
+* `Enlight_Controller_Front_RouteStartup`
+* `Enlight_Controller_Front_RouteShutdown`
+* `Enlight_Controller_Front_DispatchLoopStartup`
+* `Enlight_Controller_Front_PreDispatch`
+* `Enlight_Controller_Front_PostDispatch`
+* `Enlight_Controller_Front_DispatchLoopShutdown`
+
+
+## Controller events
+The `controller events` are, as the name implies, linked to the controllers. Each controller has certain basic methods
+which are already explained in the [controllers article](https://developers.shopware.com/developers-guide/controller/). 
+`Controller events` are defined in the `Enlight_Controller_Action` class which is located in `engine/Library/Enlight/Controller/Action.php`.
+As you can see in that file the events are generated dynamically so you are able to register on every controller's `action`, 
+`preDispatch` or `postDispatchSecure`.
+
+### preDispatch
+The `preDispatch` event will be executed before the dispatch process of a controller starts processing. 
+
+### postDispatch
+The `postDispatch` is executed after the dispatch process of the controller is finished. It is recommended 
+to use the `postDispatchSecure` event because you can be sure that this is a valid request. This event will only fired
+if a template is given. For example ajax requests and exceptions will not trigger this event.
+
+### actionEvents
+`actionEvents` will be fired everytime an action will be called. For example if the `indexAction` in the `Account` controller
+was called the event `Enlight_Controller_Action_Frontend_Account_Index` will be fired.
+
+### Event names
+The event names of the `controller events` are generated dynamically. We will use in this list placeholders for the event
+names so you are able to build the event name you need by yourself.
+
+This list is also sorted according to the time in which the events are fired. 
+
+* `Enlight_Controller_Action_PreDispatch`
```

Comment:
Move this up, and give examples of what MODULE, CONTROLLER and ACTION are. Also explain the difference between secure and non secure events